### PR TITLE
Made `now switch` and `now logs -f` work properly

### PR DESCRIPTION
**Pull request by matheuss**
This PR does 3 things:

1. Add missing `console.log`s to `now switch` – most of the `> Success!` messages weren't being printed
2. Fixes a bug in which `now switch` would never exit because the HTTP agent wasn't being closed
3. Makes the exit code (only from `now switch` – other (sub)commands needs to be adapted to `return` correctly) from subcommands bubble correctly

Before:
```
▲ ~ now switch 
> Switch to: kap
^C
```
_(I had to `ctrl+c` because it was hanging forever)_

After:
```
▲ ~ now switch    
> Switch to: kap
> Success! The team Kap is now active!
▲ ~ echo $?    
0
▲ ~ now switch 3489hrwt42           
> Error! Could not find membership for team "3489hrwt42"
▲ ~ echo $?
1
```

### CODE REVIEW

**Comment by rauchg** [comment]
3 Nits:

```
> Switch to: kap
> Success! The team Kap is now active!
```

**1** `> Switch to: kap` can be cleared

```
> Success! The team Kap is now active!
``` 

should be the consistent message for both uses

- `now switch kap`
- `now switch` (interactive)

**2** is this the case already?

**3** the success message should contain both team slug and team title

**Comment by leo** [comment]
wow, amazing! 😍

Can you also check if web socket things like deployments and `now logs -f` work just like before, because I remember those had issues with the exiting stuff we had before (which you just brought back in a similar form).

**Comment by matheuss** [comment]
@rauchg 

1. Done:
```
▲ src at fix/now-switch ✔ now switch
> Success! The team Kap is now active!
```

2. That's the case already:
```
▲ src at fix/now-switch ✔ now switch kap
> Success! The team Kap is now active!
[...]
▲ src at fix/now-switch ✔ now switch
> Success! The team Kap is now active!
```

3. Done:
```
▲ ~ now switch kap
> Success! The team Kap (kap) is now active!
```

---

@leo great catch! Adjusted `now logs -f` so it works fine with the new exit logic. `now deploy` wasn't affected 👌 I think those are the only two uses of websockets we have? 🤔

**Review by leo — APPROVED** [review]

File: src/util/input/list.js [hunk 1]
```diff
@@ -28,7 +29,8 @@ module.exports = async function({
   ],
   pageSize = 15, // Show 15 lines without scrolling (~4 credit cards)
   separator = true, // Puts a blank separator between each choice
-  abort = 'end' // Wether the `abort` option will be at the `start` or the `end`
+  abort = 'end', // Wether the `abort` option will be at the `start` or the `end`,
+  eraseFinalAnswer = false // If true, the line with the final answee that inquirer prints will be erased before returning
```
Comment (TooTallNate) [inline src/util/input/list.js:33]: `s/answee/answer`

**Pull request merged**